keps/sig-node/4188-kubelet-pod-readiness-api/README.md

```diff
@@ -217,75 +240,102 @@ the Pods. We don't want this API to return partial data.
 
 ## Design Details
 
+### Pod State Selection
+
+Kubelet maintains multiple representations of pod state:
+- The state derived from the Container Runtime Interface (CRI), reflecting the actual status of containers on the node.
+- The state that is sent to the API server, which may be subject to additional processing or delays.
+- The state received from the API server, reflecting the control plane's view.
+
+It is important to define which state this API will serve. For the purposes of this API, the intent is to serve the most
+up-to-date and accurate pod state as known locally by the Kubelet, typically the state based on the CRI and Kubelet's internal
+reconciliation, rather than the potentially stale state received from the API server. This ensures consumers receive the
+freshest possible information about pods running on the node.
+
 ### Proposed API
 
-We propose to add new gPRC API `status` in Kubelet, listening on a unix socket
+We propose to add new gRPC API `status` in Kubelet, listening on a unix socket
```

Comment:
"status" might not be the best name, since it returns more than just PodStatus.  Maybe just "pods"?